92814 E-F print tab calculate button new version performance improvement

sakshisinghjkt opened the pull request:
Updated code for EstimateCalcProcs.p for Performance improvement. It being updated to use Dataset/temp-tables instead of DB tables.

Files:
Source/est/EstimateCalcProcsAlt.p
Source/est/dsEstimate.i
Source/est/dsEstimateTT.i
Source/testers/EstimateCalcProcTester.p

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated code for EstimateCalcProcs.p for Performace improvement. It being updated to use Dataset/temp-tables instead of DB tables

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Performace improvement. Updated logic to replace table estCostForm with temp-table and then writing it into the DB in the end of processing

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Fixed discrepancy in case of Set/Combo estimates with multiple quantity cases

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Estimate calc procs optimization. Step 2- Table estCostDetail Refactor

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated program EstimateCalcProcTester.p for estCostDEtail table changes

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated code for EstimateCalcProcs.p for Performace improvement. It being updated to use temp-table EstCostSummary instead of DB table

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated code for EstimateCalcProcs.p for Performace improvement. It being updated to use temp-table instead of DB table  estCostOperations.

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated code for EstimateCalcProcs.p for Performace improvement. It being updated to use temp-table instead of DB table  estCostMisc.

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated EstCostMaterial logic for performance improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Fixed issue in EstCostMaterial Refactor

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Fixing the unit tetsing issue for estCostMaterial table

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Updated estCostMaterial winow to work on Temptable instead of DB table

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

refactoring table EstCostItem  logic and re-aligned the unique keys

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Refactored version in updated on EstimateCalcProcs.p and original version in saved as EstimateCalcProcsAlt.p

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Merged changes from latest EstimateCalcProcs.p in Develop branch

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Performance improvement- Code refactor for estCostBlank table.

Commit:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Commit:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Performance improvement- EstCostBlank table

kumarvivekjkt committed:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Refactoring for estCostHeader by using temp table and at end saving it main DB table
Files:
EstimateCalcProcs.p
estCostMaterialList.w
dsEstimateTT.i

kumarvivekjkt committed:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Renamed dsEstimatett.i to ttEstimateCalc.i and used it in EstimateCalcProcs.p and estCostMaterialList.w.
Files Committed:
est/estCostMaterialList.w
est/EstimateCalcProcs.p
est/ttEstimateCalc.i

kumarvivekjkt committed:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Done rec_key and seq assignment into the temp-table during temp-table creation and at end buffer copy temp table to DB table.
Files:
est/EstimateCalcProcs.p

brad-advantzware reviewed (changes requested):
Remove EstimateCalcProcsAlt.p - We can just keep a version of estimateCalcProcs.p in a different propath, or use the feature branch tester vs. develop/integrated test to compare outcomes.  We don't need this file.
Remove dsEstimate.i (just include the ttEstimateCalc.i)

EstimateCalcProcsTester.p should just be cleaned up if we remove EstimateCalcProcsAlt.p

kumarvivekjkt committed:
92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

Removed files EstimateCalcProcsAlt.p, dsEstimate.i. As we have removed EstimateCalcProcsAlt.p, also cleaned up testers/EstimateCalcProcTester.p.

kumarvivekjkt committed:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement

# Conflicts:
#	Source/est/EstimateCalcProcs.p
#	Source/testers/EstimateCalcProcTester.p

kumarvivekjkt committed:
Merge branch 'develop' into feature/92814_E-F_Print_Tab_Calculate_Button_New_Version_Performance_Improvement
